### PR TITLE
Fix some bugs and enhance projects page

### PR DESCRIPTION
**Pull request by Ananthavijay**
Fixes https://github.com/CircuitVerse/CircuitVerse/issues/1052

#### Describe the changes you have made in this PR -
- Fixed preview modal bug and changed the style to make it consistent.
Modal styles have been taken from this [PR](https://github.com/CircuitVerse/CircuitVerse/pull/907). Thanks @aayush-05 
- When star is clicked, view was incremented. Fixed this
- Added fork count, star count.
- I think a similar problem would exist with ````Add a collaborator```` button. Fixed this.
- Added back button

### Screenshots of the changes (If any) -
1.
Screenshot (26)

2.
Screenshot (27)

### CODE REVIEW

**Comment by Ananthavijay** [comment]
@nitin10s @tachyons @aayush-05 Please review

**Comment by yash15601** [comment]
@Ananthavijay  good work

**Comment by Ananthavijay** [comment]
should I remove the preview button then?

**Comment by aayush-05** [comment]
I agree that the preview button can be removed, since the iframe is already present on the page itself. Preview is useful on the User projects page, not necessarily on this page.

**Comment by Ananthavijay** [comment]
@nitin10s is this alright?
Screenshot (32)
Screenshot (31)

**Comment by aayush-05** [comment]
-Maintain font consistency (like the simulator button is having a different font)
-Use SVG files from prototype for icons
-Make them float to the right (also confirm if we are keeping a count for forks, can't see it in the old UI"
image

**Comment by Ananthavijay** [comment]
The prototype doesnt have these icons

**Comment by tachyons** [comment]
@satu0king

**Comment by Ananthavijay** [comment]
I think the font is the same. And in the old UI fork count is missing so removed it and aligned the view count and star count right. Should we need the fork count?

**Comment by Ananthavijay** [comment]
Screenshot:
Screenshot (33)

**Comment by yash15601** [comment]
Well working 🙂

**Comment by Ananthavijay** [comment]
@nitin10s @aayush-05 Please review

**Comment by Shivansh2407** [comment]
@nitin10s and @tachyons please review and merge this so that I can open a PR for #1091 . I am ready with the solution of the issue 👍

**Comment by aayush-05** [comment]
> I think the font is the same. And in the old UI fork count is missing so removed it and aligned the view count and star count right. Should we need the fork count?

image
The fonts on these buttons look inconsistent to me.
Also can you move to the inline styles to the respective stylesheets

**Comment by Ananthavijay** [comment]
@aayush-05 is this okay?
Screenshot (35)

**Comment by yash15601** [comment]
@Ananthavijay LGTM👀

**Comment by Ananthavijay** [comment]
And also moving inline styles to the respective stylesheets will be done by @Shivansh2407 because he has a fix to an issue which is dependent on this PR

**Comment by aayush-05** [comment]
Can you post the screenshots for Iphone 5/SE display as well ? @Ananthavijay

**Comment by Ananthavijay** [comment]
Sure 
Screenshot (42)

File: app/assets/stylesheets/logixRelease.scss [hunk 1]
```diff
@@ -336,7 +336,7 @@ a.btn.btn-outline-danger {
 }
 
 .fa-post {
-  font: normal normal normal 14px/1 FontAwesome;
+  font: normal normal normal 14px/1 sans-serif;
```
Comment (tachyons) [inline app/assets/stylesheets/logixRelease.scss]: Curios why?

Comment (Ananthavijay) [reply app/assets/stylesheets/logixRelease.scss]: I tried to change the font but there were alignment issues. So changed it like this

**Pull request merged**